README.md

```diff
@@ -37,7 +44,7 @@ other region like province), or country that they belong to. Pelias has the
 ability to compute these values from [Who's on First](http://whosonfirst.mapzen.com/) data.
 For more info on how admin lookup works, see the documentation for
 [pelias/wof-admin-lookup](https://github.com/pelias/wof-admin-lookup). By default,
-adminLookup is enabled.  To disable, set `imports.adminLookup.enabled` to `false` in Pelias config.
+adminLookup is enabled.  To disable, set `imports.openaddresses.adminLookup` to `false` in Pelias config.
```

Comment:
If I remember correctly, we moved this flag, it's no longer per-importer, so the readme is correct.